test/integration/script/script.txtar

```diff
@@ -0,0 +1,41 @@
+exists x.wasm
```

Comment:
these "exists" checks might not be necessary now that the test is setup. They were helpful sanity checks in development.

Reply:
removed these exists checks